app/src/main/java/org/ole/planet/myplanet/datamanager/Service.kt

```diff
@@ -295,9 +295,10 @@ class Service(private val context: Context) {
             override fun onResponse(call: Call<JsonObject?>, response: Response<JsonObject?>) {
                 if (response.isSuccessful) {
                     response.body()?.let { jsonObject ->
-                        val currentVersion = "${context.resources.getText(R.string.app_version)}"
+                        val rawCurrentVersion = "${context.resources.getText(R.string.app_version)}"
+                        val cleanCurrentVersion = rawCurrentVersion.replace("-lite", "")
                         val minApkVersion = jsonObject.get("minapk").asString
-                        if (isVersionAllowed(currentVersion, minApkVersion)) {
+                        if (isVersionAllowed(cleanCurrentVersion, minApkVersion)) {
```

Comment:
Since the root of the issue is calling `isVersionAllowed`, which calls `compareVersions`, would it make more sense to make changes in `compareVersions`? There's already `removePrefix("v")`, so we can just add `.removeSuffix("-lite")` behind it?